cmd/cdi-controller/controller.go

```diff
@@ -193,9 +201,25 @@ func main() {
 	}
 
 	<-stopCh
+
+	deleteReadyFile()
+
 	glog.V(2).Infoln("cdi controller exited")
 }
 
+func createReadyFile() error {
+	f, err := os.Create(readyFile)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	return nil
+}
+
+func deleteReadyFile() {
+	os.Remove(readyFile)
```

Comment:
Do we want to check for an error on the delete, not sure what we would do with the error if it happens. But not being able to delete the file is probably a problem.

Reply:
Useless info IMO.  Will just make code uglier and decrease coverage numbers.

Reply:
I missed this is basically the last thing before exit, can't even do anything useful with an error message if we got one.